### PR TITLE
feat: add DriveReady

### PR DESCRIPTION
**Pull request by SpaceEnergy**

### CODE REVIEW

**Comment by alistair3149** [comment]
Should we do a drive ready? CIG uses flight ready for ground vehicles as well.
Originally we are using the scheme provided here: https://support.robertsspaceindustries.com/hc/en-us/articles/360003093114-Loaner-Ship-Matrix

**Comment by SpaceEnergy** [comment]
at .wiki we use for ground vehicles driveready

**Comment by octfx** [comment]
If .tools is fine with it so am I

**Comment by alistair3149** [comment]
> If .tools is fine with it so am I

It doesn't affect us. We will probably stick to flight ready for now unless CIG changes the status.

File: Vehicle/i18n.json [hunk 1]
```diff
@@ -842,6 +842,13 @@
 				"en": "Flight ready"
 			}
 		],
+		[
+			"DriveRead",
```
Comment (octfx) [inline Vehicle/i18n.json]: There seems to be a `y` missing

**Review by octfx — CHANGES_REQUESTED** [review]

**Comment by SpaceEnergy** [comment]
fixed :D

**Pull request merged**